converters/toarray.ts

```diff
@@ -0,0 +1,26 @@
+import DiscoverConverters from "./toarray/discover";
+import InstrumentationConverters from "./toarray/instrumentation";
+import LayoutConverters from "./toarray/layout";
+import PerformanceConverters from "./toarray/performance";
+import PointerConverters from "./toarray/pointer";
+import ViewportConverters from "./toarray/viewport";
+
+export default function(event: IEvent): IEventArray {
```

Comment:
>function(event: IEvent): IEventArray [](start = 15, length = 36)

I would prefer to have an interface /serializer. Something like:

IEventSerializer<TIn,Tout> IEventSerializer
{
    TOut  serialize(TIn input)
   TIn deserailize(TOut output)
}

Seems cleaner and easier to understand. But keep the binding very high level.